diff --git a/node_modules/expo-firebase-core/node_modules/expo-constants/android/build.gradle b/node_modules/expo-firebase-core/node_modules/expo-constants/android/build.gradle
index 744c94b..bc0e9cf 100644
--- a/node_modules/expo-firebase-core/node_modules/expo-constants/android/build.gradle
+++ b/node_modules/expo-firebase-core/node_modules/expo-constants/android/build.gradle
@@ -37,19 +37,11 @@ buildscript {
   }
 }
 
-// Creating sources with comments
-task androidSourcesJar(type: Jar) {
-  classifier = 'sources'
-  from android.sourceSets.main.java.srcDirs
-}
-
 afterEvaluate {
   publishing {
     publications {
       release(MavenPublication) {
         from components.release
-        // Add additional sourcesJar to artifacts
-        artifact(androidSourcesJar)
       }
     }
     repositories {
@@ -61,6 +53,7 @@ afterEvaluate {
 }
 
 android {
+  namespace = "expo.modules.constants"
   compileSdkVersion safeExtGet("compileSdkVersion", 31)
 
   compileOptions {
@@ -81,6 +74,11 @@ android {
   lintOptions {
     abortOnError false
   }
+  publishing {
+    singleVariant("release") {
+      withSourcesJar()
+    }
+  }
 }
 
 repositories {
diff --git a/node_modules/expo-firebase-core/node_modules/expo-constants/android/src/main/AndroidManifest.xml b/node_modules/expo-firebase-core/node_modules/expo-constants/android/src/main/AndroidManifest.xml
index b5b338c..4b45e77 100644
--- a/node_modules/expo-firebase-core/node_modules/expo-constants/android/src/main/AndroidManifest.xml
+++ b/node_modules/expo-firebase-core/node_modules/expo-constants/android/src/main/AndroidManifest.xml
@@ -1,5 +1,5 @@
 
-<manifest package="expo.modules.constants">
+<manifest>
 
 </manifest>
   
